ui/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
         versionCode cfg.versionCode
         versionName cfg.versionName
         buildConfigField "String", "main_app", '"'+cfg.package+'"'
+        vectorDrawables.useSupportLibrary = true
```

Comment:
Just like with dagger comment, I think you're using vectorDrawables in your project and you need this. Please if this is something that only affects your project, can you remove it from here? Thanks!

Reply:
When updating, you are asked to "add google maven repository and sync project," after doing so, without the above line, a wall of these errors occurs:

dhis2-android-sdk/ui/src/main/res/drawable/ic_arrow_back.xml : Can't process attribute android:fillColor="@android:color/white": references to other resources are not supported by build-time PNG generation. See http://developer.android.com/tools/help/vector-asset-studio.html for details.

This occurs with the official dhis2-tracker-capture and not our local project

Reply:
@Ezmeyers you're right. This is needed to fully complete the upgrade keeping backward compatibility to some properties that we're using in some xmls. I'm accepting this change. BTW, following your PR we've decided to upgrade versions of most of the libraries we're using. We'll introduce it for the next version. Thanks for the contribution!